### PR TITLE
Quickstart: Run PyTorch Inference on Intel GPU using Docker (on Linux or WSL)

### PR DESCRIPTION
**Pull request by liu-shaojun**
### 1. Why the change?

- Add a quickstart to run PyTorch Inference on Intel GPU using Docker
- Update ipex-llm-xpu image to add benchmark & examples 

You may check http://10.239.44.83:8008/doc/LLM/Quickstart/docker_pytorch_inference_gpu.html to see the quickstart.

Perf: https://github.com/analytics-zoo/nano/issues/1367

### CODE REVIEW

**Comment by jason-dai** [comment]
We just need one quick start for running PyTorch inference on GPU using Docker (on Linux or WSL):
1) Install docker
2) Launch docker
3) Run inference benchmark
4) Run chat.py?
5) Run PyTorch examples

**Comment by liu-shaojun** [comment]
> We just need one quick start for running PyTorch inference on GPU using Docker (on Linux or WSL):
> 
> 1. Install docker
> 2. Launch docker
> 3. Run inference benchmark
> 4. Run chat.py?
> 5. Run PyTorch examples

Updated

File: docs/readthedocs/source/doc/LLM/Quickstart/docker_pytorch_inference_gpu.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+```eval_rst
+.. note::
+
+  If you want to benchmark the performance without warmup, you can set ``warm_up: 0`` and ``num_trials: 1`` in ``config.yaml``, and run each single model and in_out_pair separately. 
+```
+
+
+After configuring the `config.yaml`, run the following scripts:
+```bash
+source ipex-llm-init -g --device <value>
```
Comment (glorysdj) [inline docs/readthedocs/source/doc/LLM/Quickstart/docker_pytorch_inference_gpu.md:139]: --gpu？

Comment (glorysdj) [inline docs/readthedocs/source/doc/LLM/Quickstart/docker_pytorch_inference_gpu.md:139]: --gpu？

File: docker/llm/README.md [hunk 2]
```diff
@@ -159,9 +159,12 @@ Run the following command to pull image from dockerhub:
 docker pull intelanalytics/ipex-llm-xpu:2.1.0-SNAPSHOT
```
Comment (shane-huang) [inline docker/llm/README.md:159]: readme seems too long and may be simplified.

**Review by shane-huang — APPROVED** [review]
I think we can merge this version and revise in following PRs.

**Pull request merged**